applications/body_pose_estimation/CMakeLists.txt

```diff
@@ -12,34 +12,44 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+project(body_pose_estimation NONE)
 
 find_package(holoscan 1.0 REQUIRED CONFIG
              PATHS "/opt/nvidia/holoscan" "/workspace/holoscan-sdk/install")
 
 # Download the yolov8 pose model if it doesn't exist
-if(NOT EXISTS "${HOLOHUB_DATA_DIR}/body_pose_estimation/yolov8l-pose.onnx")
-    add_custom_target(yolov8_onnx ALL
-      COMMAND yolo export model=yolov8l-pose.pt format=onnx
-      COMMAND mkdir -p "${HOLOHUB_DATA_DIR}/body_pose_estimation"
-      COMMAND mv yolov8l-pose.onnx "${HOLOHUB_DATA_DIR}/body_pose_estimation"
-      BYPRODUCTS "yolov8l-pose.onnx"
+add_custom_command(OUTPUT "${HOLOHUB_DATA_DIR}/body_pose_estimation/yolov8l-pose.onnx"
```

Comment:
@jjomier what's the difference between the usage for `OUTPUT` and `BYPRODUCTS`?

Reply:
The `OUTPUT` is used by CMake to find dependencies between the command.
The `BYPRODUCTS` allows to remove the datasets when using make clean